### PR TITLE
12 get article by id with comment

### PR DESCRIPTION
**Pull request by WanwanShi**
test- add new 200 success test for/api/articles/:article_id   comment count of 0 and non-zero.

added the function that adds comment_count property to the returned object

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -81,6 +81,50 @@ describe("/api/articles/:article_id", () => {
...
+                    body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+                    created_at: expect.any(String),
+                    article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                    votes: 0,
+                    comment_count: 0
+                })
+            )
+        })
+    })
+
```
Comment (nc-niamh-smith) [inline __tests__/app.test.js:127]: 🎉 Works well Wendy!

File: models/articles_models.js [hunk 2]
```diff
@@ -2,9 +2,15 @@ const db = require("../db/connection")
...
-                    WHERE article_id = $1; 
-                    `,[article_id])
+    const sqlString = `SELECT  articles.article_id, articles.title, articles.author,articles.topic,articles.body, articles.created_at,articles.votes,articles.article_img_url, COUNT(comments.body)::INT AS  comment_count
+    FROM articles
+    LEFT JOIN comments 
+    ON articles.article_id = comments.article_id
+    WHERE articles.article_id = $1
+    GROUP BY articles.article_id;`
+
+    return db.query(sqlString
```
Comment (nc-niamh-smith) [inline models/articles_models.js:12]: Lovely, simple alteration. Well done :)

**Review by nc-niamh-smith — COMMENTED** [review]
Really nice implementation of the feature Wendy, well done 🎉
